batterymonitor@pdcurtis/files/batterymonitor@pdcurtis/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "max-instances": "1",
-    "description": "Displays Charge as Percentage and allows Alerts and Actions",
+    "description": "Displays Charge as Percentage and allows Alerts and Actions\n Yaru icon theme (C) Canonical distributed under the CC-BY-SA 4.0 license\n https://creativecommons.org/licenses/by-sa/4.0/",
```

Comment:
The attribution can be appended to the README. That is cleaner but also avoids the changes to translations since this field will be picked up.